public/app/features/variables/editor/ConfirmDeleteModal.tsx

```diff
@@ -0,0 +1,34 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { ConfirmModal } from '@grafana/ui';
+
+interface Props {
+  varName: string;
+  isOpen: boolean;
+  onConfirm: () => void;
+  onDismiss: () => void;
+}
+
+export function ConfirmDeleteModal({ varName, isOpen = false, onConfirm, onDismiss }: Props) {
+  return (
+    <ConfirmModal
+      title="Delete"
```

Comment:
```suggestion
      title="Delete variable"
```

Nit: more clarity to the users in where they are